build.cake

```diff
@@ -402,6 +404,143 @@ Task("PackageConsole")
         });
     });
 
+Task("PackageChocolatey")
+	.Description("Creates chocolate packages of the console runner")
+    .IsDependentOn("CreateImage")
+	.Does(() =>
+	{
+		var currentImageDir = IMAGE_DIR + "NUnit-" + packageVersion + "/";
+		
+		EnsureDirectoryExists(PACKAGE_DIR);
+		
+		// Note: Since cake does not yet support a working directory and separate output directory for chocolatey, the following copying and hacks are needed.
+		
```

Comment:
@Roemer This came from your branch and I just kept it. I actually don't understand it either.